.winget/configuration.dsc.yaml

```diff
@@ -57,6 +57,14 @@ properties:
       settings:
         id: Microsoft.PowerShell
         source: winget
+    - resource: Microsoft.WinGet.DSC/WinGetPackage
```

Comment:
Microsoft [made Terminal an inbox app](https://blogs.windows.com/windows-insider/2021/03/17/announcing-windows-10-insider-preview-build-21337/#:~:text=Windows%20Terminal%20now%20included%20as%20an%20inbox%20app), so it should already be on modern systems, but good to check and DSC does exactly that - thanks for adding this back.